server/src/test/java/org/cloudfoundry/identity/uaa/provider/token/JwtBearerAssertionTokenAuthenticatorTest.java

```diff
@@ -176,43 +211,43 @@ public void testTokenWithNoExpClaim() {
         String token = testTokenUtil.mockInvalidExpirationAssertionToken(DEVICE_1_ID, 0, TENANT_ID,
                 AUDIENCE, null);
         this.tokenAuthenticator.setClientDetailsService(this.clientDetailsService);
-        this.tokenAuthenticator.authenticate(token);
+        this.tokenAuthenticator.authenticateWithoutClientAssertionHeader(token);
     }
 
```

Comment:
Should we test for an assertion token with missing (null or empty) subject and issuer claims as well?